<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>LandMarkExtractView</class>
 <widget class="QWidget" name="LandMarkExtractView">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>403</width>
    <height>468</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QWidget" name="verticalLayoutWidget">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>381</width>
     <height>141</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QLabel" name="label">
        <property name="text">
         <string>Image:</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QmitkDataStorageComboBox" name="ImageSelector"/>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_2">
      <item>
       <widget class="QLabel" name="label_2">
        <property name="text">
         <string>Diameter:</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="DiameterLe">
        <property name="text">
         <string>3.0</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="label_3">
        <property name="text">
         <string>mm</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <widget class="ctkRangeWidget" name="ThresholdSlider" native="true"/>
    </item>
    <item>
     <widget class="QPushButton" name="ExtractBtn">
      <property name="text">
       <string>Extract</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QmitkPointListWidget" name="PointListWidget" native="true">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>160</y>
     <width>379</width>
     <height>251</height>
    </rect>
   </property>
  </widget>
 </widget>
 <customwidgets>
  <customwidget>
   <class>QmitkDataStorageComboBox</class>
   <extends>QComboBox</extends>
   <header>QmitkDataStorageComboBox.h</header>
  </customwidget>
  <customwidget>
   <class>QmitkPointListWidget</class>
   <extends>QWidget</extends>
   <header>QmitkPointListWidget.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>ctkRangeWidget</class>
   <extends>QWidget</extends>
   <header>ctkRangeWidget.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
